ttrpc-codegen/Cargo.toml

```diff
@@ -16,4 +16,4 @@ readme = "README.md"
 protobuf = { version = "2.14.0" }
 protobuf-codegen-pure = "2.14.0"
 protobuf-codegen = "2.14.0"
-ttrpc-compiler = "0.4.0"
+ttrpc-compiler = { path = "../compiler" }
```

Comment:
It will block `cargo publish` of `ttrpc-codegen`, can you split this pr to two, the first one update ttrpc-compiler, the 2nd one update the ttrpc and examples, and we will publish the new version of ttrpc-compiler and ttrpc-codegen after the 1st pr before the 2nd one.

Reply:
OK, do I need to update the version numbers?

Reply:
I have done this already.